service/index.js

```diff
@@ -171,8 +160,6 @@ app.get(/^\/v1\/polyfill(\.\w+)(\.\w+)?/, function(req, res) {
 	res.set('Content-Type', contentTypes[fileExtension]+';charset=utf-8');
 	res.set('Access-Control-Allow-Origin', '*');
 	res.send(op);
-	metrics.addResponseTime(Date.now() - responseStartTime);
```

Comment:
Where's the current data?  If it's based on the stuff in Heroku it's more likely to be how fast node responds to the `__gtg`, which v8 will have JIT'd for us.  I still think measuring this here makes sense.